src/gui/SquareSvgWidget.h

```diff
@@ -18,16 +18,20 @@
 #ifndef KEEPASSX_SquareSvgWidget_H
 #define KEEPASSX_SquareSvgWidget_H
 
+#include <QLayout>
 #include <QSvgWidget>
 
 class SquareSvgWidget : public QSvgWidget
 {
 public:
-    SquareSvgWidget() = default;
+    explicit SquareSvgWidget(QWidget* parent)
+        : QSvgWidget(parent){};
```

Comment:
Our convention is to have the constructor definition in the cpp file

Reply:
Sure, thank you, fixed.